types/node/process.d.ts

```diff
@@ -1304,6 +1304,23 @@ declare module "process" {
                  */
                 uptime(): number;
                 hrtime: HRTime;
+                /**
+                 * If the Node.js process was spawned with an IPC channel, the process.channel property is a reference to the IPC channel.
+                 * If no IPC channel exists, this property is undefined.
+                 * @since v7.1.0
+                 */
+                channel?: {
```

Comment:
Shouldn't it have the same type as https://nodejs.org/dist/latest-v20.x/docs/api/child_process.html#subprocesschannel ?

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0904159479fa886e08cf323ec4d15dadb44355b4/types/node/child_process.d.ts#L137-L142

Reply:
I didn't use `Pipe` because it seems to have the following type:

```ts
interface Pipe {
    close(): void;
    hasRef(): boolean;
    ref(): void;
    unref(): void;
}
```

which didn't match up with the `process.channel` documentation:
https://nodejs.org/dist/latest-v20.x/docs/api/process.html#processchannel